### ISSUE TITLE
[HELP] Mount Path Does Not Exist (it does)

### ISSUE THREAD

**Issue by AnCasper** [rogerfar/rdt-client#562]
**What version are you using?**
v2.0.81

**Wat OS are you running?**
WIndows

**Are you using Docker or as a service?**
Service

**Which debrid provider are you using?**
Real-Debrid

**Which downloader are you using?**
Symlink Downloader

**Please attach a log file here with the log setting set to debug**
What you see in the images is the only relating errors.

**Mount does exist and is accesable: RDT running as Admin, just to be sure. Flasger are set as follows: --dir-cache-time 30s --allow-other --network-mode**

![Screenshot 2024-08-09 195522](https://github.com/user-attachments/assets/d6479efe-f92d-4c2f-9ed9-f6cd99912b6e)

![Screenshot 2024-08-09 195619](https://github.com/user-attachments/assets/19b739d8-939f-4c1a-b202-b2d1d7f00882)

![Screenshot 2024-08-09 195959](https://github.com/user-attachments/assets/9b979923-08b8-46b9-9946-442da2898541)

Should also note that there is also an issue when it comes to Symlinking the files. RDT try's to sym files that do not exist in the root of the mounted rclone drive, for example: 
Trys: `Z:\Fast and Furious.mkv`
Instead of: `Z\:__all__\<FoldeName>\<FileName>`

**Comment by AnCasper** [comment]
Update: managed to get this error - `[12:38:35 ERR] Error creating symbolic link from Z:\__all__\Fast And Furious - Solo Parti Originali (2009) 2160p H265 10 bit ita eng AC3 5.1 sub ita eng Licdom.mkv\Fast And Furious - Solo Parti Originali (2009) 2160p H265 10 bit ita eng AC3 5.1 sub ita eng Licdom.mkv to H:\Sruff\Jellyfin\Downloads\Movies\Fast And Furious - Solo Parti Originali (2009) 2160p H265 10 bit ita eng AC3 5.1 sub ita eng Licdom.mkv\Fast And Furious - Solo Parti Originali (2009) 2160p H265 10 bit ita eng AC3 5.1 sub ita eng Licdom.mkv: Incorrect function. : 'H:\Sruff\Jellyfin\Downloads\Movies\Fast And Furious - Solo Parti Originali (2009) 2160p H265 10 bit ita eng AC3 5.1 sub ita eng Licdom.mkv\Fast And Furious - Solo Parti Originali (2009) 2160p H265 10 bit ita eng AC3 5.1 sub ita eng Licdom.mkv'
`
Note the: "Incorrect function." part.

**Comment by AnCasper** [comment]
[Log](https://api.mclo.gs/1/raw/DnxuYaJ)

**Comment by AnCasper** [comment]
Looks like RDT Client does not handle files inside folders from the mount. Seems to be adding **`.mkv (or any file extension)`** to the end of a folders file name making RDT Client throw an error.

Using: **`retain_folder_name_extension: true`** on rclone does not solve the problem either.

**FROM** and **TO** have different pathing structure:
`2024-08-11 14:29:59.838 +01:00 [DBG] RdtClient.Service.Services.Downloaders.SymlinkDownloader: Creating symbolic link **FROM** Z:\Movies\Fast.Five.2011.EXTENDED.2160p.BluRay.x265.10bit.HDR.DTS-X.7.1-SWTYBLZ\Fast.Five.2011.EXTENDED.2160p.BluRay.x265.10bit.HDR.DTS-X.7.1-SWTYBLZ.mkv **TO** H:\Sruff\Jellyfin\Movies\Fast.Five.2011.EXTENDED.2160p.BluRay.x265.10bit.HDR.DTS-X.7.1-SWTYBLZ.mkv\Fast.Five.2011.EXTENDED.2160p.BluRay.x265.10bit.HDR.DTS-X.7.1-SWTYBLZ.mkv`
`2024-08-11 14:29:59.844 +01:00 [ERR] RdtClient.Service.Services.Downloaders.SymlinkDownloader: Error creating symbolic link from Z:\Movies\Fast.Five.2011.EXTENDED.2160p.BluRay.x265.10bit.HDR.DTS-X.7.1-SWTYBLZ\Fast.Five.2011.EXTENDED.2160p.BluRay.x265.10bit.HDR.DTS-X.7.1-SWTYBLZ.mkv to H:\Sruff\Jellyfin\Movies\Fast.Five.2011.EXTENDED.2160p.BluRay.x265.10bit.HDR.DTS-X.7.1-SWTYBLZ.mkv\Fast.Five.2011.EXTENDED.2160p.BluRay.x265.10bit.HDR.DTS-X.7.1-SWTYBLZ.mkv: Incorrect function. : 'H:\Sruff\Jellyfin\Movies\Fast.Five.2011.EXTENDED.2160p.BluRay.x265.10bit.HDR.DTS-X.7.1-SWTYBLZ.mkv\Fast.Five.2011.EXTENDED.2160p.BluRay.x265.10bit.HDR.DTS-X.7.1-SWTYBLZ.mkv'`

Also again with **`Incorrect function`**, just a bit vague.

**Comment by Carlosjrlu** [comment]
Hey sorry to bother you, but I'm in the same boat with this error. Did you manage to solve it? And how?

**Comment by AnCasper** [comment]
> Hey sorry to bother you, but I'm in the same boat with this error. Did you manage to solve it? And how?

Yes, for me it was because my drive was formated to exFAT, I meant to post the solution to my issue but have been busy.

Hopefully this solves your problem as it did mine :)
